content/id/docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -0,0 +1,234 @@
+`.spec.nodeName`. Kemudian, _default scheduler_ digunakan untuk mengikat pod ke
+host target. Jika afinitas node dari pod DaemonSet sudah ada, maka ini
+akan diganti. _Controller DaemonSet_ hanya akan melakukan operasi-operasi ini
+ketika membuat atau mengubah pod DaemonSet, dan tidak ada perubahan yang terjadi
+pada `spec.template` DaemonSet.
+
+```yaml
+nodeAffinity:
+  requiredDuringSchedulingIgnoredDuringExecution:
+    nodeSelectorTerms:
+    - matchFields:
+      - key: metadata.name
+        operator: In
+        values:
+        - target-host-name
+```
+
+Sebagai tambahan, _toleration_ `node.kubernetes.io/unschedulable:NoSchedule`
+ditambahkan secara otomatis pada pod DaemonSet. _Default scheduler_ akan
+mengabaikan node `unschedulable` ketika menjadwalkan pod DaemonSet.
+
+### _Taint_ dan _Toleration_
+
+Meskipun Pod Daemon menghormati
+[taint dan toleration](/docs/concepts/configuration/taint-and-toleration),
+_toleration_ berikut ini akan otomatis ditambahkan ke Pod DaemonSet sesuai
+dengan fitur yang bersangkutan.
+
+| _Toleration Key_                         | _Effect_   | Versi   | Deskripsi                                                                                                    |
+| ---------------------------------------- | ---------- | ------- | ------------------------------------------------------------------------------------------------------------ |
+| `node.kubernetes.io/not-ready`           | NoExecute  | 1.13+   | Pod DaemonSet tidak akan menjadi _evicted_ ketika ada masalah node seperti partisi jaringan.                 |
+| `node.kubernetes.io/unreachable`         | NoExecute  | 1.13+   | Pod DaemonSet tidak akan menjadi _evicted_ ketika ada masalah node seperti partisi jaringan.                 |
+| `node.kubernetes.io/disk-pressure`       | NoSchedule | 1.8+    |                                                                                                              |
+| `node.kubernetes.io/memory-pressure`     | NoSchedule | 1.8+    |                                                                                                              |
+| `node.kubernetes.io/unschedulable`       | NoSchedule | 1.12+   | Pod DaemonSet mentoleransi atribut `unschedulable` _default scheduler_.                                      |
+| `node.kubernetes.io/network-unavailable` | NoSchedule | 1.12+   | Pod DaemonSet yang menggunakan jaringan host mentoleransi atribut `network-unavailable` _default scheduler_. |
+
+
+
+## Berkomunikasi dengan Pod Daemon
+
+Beberapa pola yang mungkin digunakan untuk berkomunikasi dengan Pod dalam DaemonSet, yaitu:
+
+- **Push**: Pod dalam DaemonSet diatur untuk mengirim pembaruan status ke servis lain,
+  contohnya _stats database_. Pod ini tidak memiliki klien.
+- **NodeIP and Known Port**: Pod dalam DaemonSet dapat menggunakan `hostPort`, sehingga Pod dapat diakses menggunakan IP node. Klien tahu daftar IP node dengan suatu cara, dan tahu port berdasarkan konvensi.
```

Comment:
In this line, “and” is untranslated. (Also “known”, but that might be intentional?)
```suggestion
- **NodeIP dan Known Port**: Pod dalam DaemonSet dapat menggunakan `hostPort`, sehingga Pod dapat diakses menggunakan IP node. Klien tahu daftar IP node dengan suatu cara, dan tahu port berdasarkan konvensi.
```

Reply:
"IP Node dan Konvensi Port"
sounds good to me. 
it's not referring to API Object and "konvensi port" already used on description

Reply:
Cool. I'll add it then.